src/containers/AccountModal/AccountModal.tsx

```diff
@@ -73,11 +72,10 @@ const AccountModal = () => {
         </div>
       );
     }
+
     switch (view) {
-      case 'simultaneous-logins':
-        return <SimultaneousLoginsNotification />;
       case 'login':
-        return <Login />;
+        return <Login message={message} />;
```

Comment:
Good that you aren't passing the message to display in full in the query param, but I would suggest making message into an enum that the Login component understands so it can be expanded if needed (i.e. instead of "true" use "simultaneous_warning" or something.) We can parse the value here and have the props on the components be enum values, so it's more intuitive.

Reply:
In the paymentFailed component it is approached in the same way see below: 

![Screenshot 2023-07-03 at 13 18 58](https://github.com/jwplayer/ott-web-app/assets/104987342/7d793589-f493-476f-b5d2-1000ecd1a50b)
![Screenshot 2023-07-03 at 13 18 40](https://github.com/jwplayer/ott-web-app/assets/104987342/eebe57cd-5546-4b6b-b1a5-61f849bf609c)

Should i also refactor this component ?

Reply:
Yes if you have time to do that it would be great, otherwise, at least make sure the new code you are adding is good